js/components/app.js

```diff
@@ -4,35 +4,87 @@ ScreenAddContact
 ScreenGroupDetails
 ScreenGroupList
 ScreenFeedsAndCreatePost
+ScreenCreatePost
 */
 
 window.App = React.createClass({
   getInitialState: function () {
     return {
-      route: "GroupDetails"
+      route: "GroupList",
+      groupId: "abc",
+      posts: this.props.jsonPosts
     };
   },
-
   setRoute(r) {
     this.setState({ route: r });
   },
-
-
+  setGroupId(i) {
+    this.setState({ groupId: i });
+  },
+  getContacts: function () {
+    return this.props.jsonContacts;
```

Comment:
Hmm. Okay. Leave this as it is. I have different idea but we will do it when we get to using an actual API